src/main/resources/kfsdbupgrade.properties

```diff
@@ -67,7 +67,7 @@ files-5.3.2_5.4=rice_server/rice-server-script.xml,rice_server/kew_upgrade.xml,r
+files-post-upgrade=sql/kim_to_ldap_mapping_params.sql,sql/missing_components.sql,sql/kim_updates.sql,sql/liquibase_upgrade.sql,sql/ca_account_ext_t_migration.sql,sql/UAF-2043_update_phase_2_parameters.sql,sql/add_back_parameters_deleted.sql,sql/add_lookup_records_permission.sql,sql/inactivate_role_11053.sql,sql/update_system_enhancement_parameters.sql,sql/update_requisition_source.sql,sql/UAF-2739_add_responsibility_49_role_10403.sql,sql/update_disbursement_voucher_payment_reason.sql,sql/UAF-2377-PopulateW8Type.sql,sql/UAF-2768_update_item_type_maintenance_table.sql,sql/UAF-2835-Phase1Parameters.sql,sql/UAF-2907-UpdateSystemParameters.sql,sql/UAF-2911-InactivateConflictingPermission.sql,sql/UAF-2688-AddParametersForOLTPLockingEnhancement.sql,sql/UAF-2875_remove_permission_vendor_exclude_upload.sql,sql/UAF-2691_delete_scrapped_3_parameters.sql,sql/UAF-2893_inactivate_vendor_type_refund_vendor.sql,sql/UAF-2782_update_method_PO_transmission_lookup.sql,sql/UAF-2711_update_parameter_SENSITIVE_DATA_PATTERNS.sql,sql/UAF-2854-Phase3Parameters.sql,sql/UAF-95-CreateFSOWebPageForPOTermsAndConditions.sql,sql/UAF-2549_update_ST_YEST_parameters.sql,sql/UAF-3012_activate_role_24_assign_responsibility_3.sql,sql/UAF-3016_delete_permissions_discontinued.sql,sql/UAF-3060_update_purchase_order_contract_part_2.sql,sql/UAF-3427_Update_Parameter_OBJECT_TYPE_BYPASS_ORIGINATIONS.sql,sql/UAF-2802_Add_PCDM_Permission_And_Responsibility.sql,sql/UAF-3618_add_parm_values_to_DB_upgrade.sql,sql/UAF-3671_Update_Namespace_KFS-PCARD_AutoApprove.sql,sql/UAF-3846_Remove_Retired_Responsibilities.sql,sql/UAF-3856-Update_namespace_for_Batch_view_perm.sql,sql/UAF-4059_parameters_chart_code_autofill.sql,sql/UAF-4180_Update_NUMBER_OF_DAYS_USED_TO_DECREASE_CALCULATED_PAY_DATE.sql,sql/UAF-4183_Delete_Scrapped_ENABLE_APO_IND.sql,sql/UAF-4294_Update_pcard_pk.sql,sql/UAF-31_Check_Reconciliation_Database_Update.sql,sql/UAF-40_Add_component_DocumentCreationStep.sql,sql/UAF-4877.sql,sql/UAF-4199.sql
```

Comment:
This property is inconsequential in this baseline -- there is a java file filter that matches against the post-upgrade dir for any files with an sql extension, and uses it if found.